pages/download.py

```diff
@@ -133,8 +133,8 @@ def get_content(self):
 
 Freenet requires OS X 10.8 or later.
 """) + "\n\n" + """
-[url_mac_installer]: https://github.com/freenet/mactray/releases/download/v2.0.3/FreenetTray_2.0.3.zip
-[url_gpg_sig]: https://downloads.freenetproject.org/alpha/mactray/FreenetTray_2.0.3.zip.sig
+[url_mac_installer]: https://github.com/freenet/mactray/releases/download/v2.0.4/FreenetTray_2.0.4.zip
+[url_gpg_sig]: https://downloads.freenetproject.org/alpha/mactray/FreenetTray_2.0.4.zip.sig
```

Comment:
I think @Thynix goal was to offload the rather large download to GitHub but still keep the `.sig` on the server to have a bit more control over what was signed/linked from the website, not sure.